packages/beacon-node/package.json

```diff
@@ -98,6 +98,7 @@
     "@chainsafe/as-chacha20poly1305": "^0.1.0",
     "@chainsafe/as-sha256": "^0.3.1",
     "@chainsafe/bls": "7.1.1",
+    "@chainsafe/blst": "file:../../../blst-ts-2/rebuild",
```

Comment:
@dapplion I have the new bindings stubbed in using a file import.  How would you prefer I handle this for CI?  Should I publish a version to `@matthewkeil/blst` or something similar? Or perhaps an odd semVer number that we can delete from `@chainsage/blst`?

Reply:
I published at `blst-ts-test` for now